bin/nomplate.js

```diff
@@ -11,7 +11,13 @@ function printHelp() {
 }
 
 function printError(message) {
-  console.error(message);
+  if (String(message).toLowerCase().includes('unexpected token import')) {
+    console.log('Nomplate does not support ES6-style "import" for modules.\n\
+    Please use "require" statements instead.\n\
+    e.g. const nomplate = require("nomplate")');
+  } else {
```

Comment:
These shouldn't be mutually exclusive, just print our message above the other and folks will still get the stack trace so they can find the offending line.